vendor/github.com/containerd/nydus-snapshotter/pkg/converter/constant.go

```diff
@@ -8,6 +8,7 @@ package converter
 
 const (
 	ManifestOSFeatureNydus   = "nydus.remoteimage.v1"
+	ManifestConfigNydus      = "application/vnd.nydus.image.config.v1+json"
```

Comment:
This was added in this PR;

- https://github.com/containerd/nydus-snapshotter/pull/674

@imeoer is there anything to do on the BuildKit side for this, or no need to wire this up?